packages/test/mocha-test-setup/src/mochaHooks.ts

```diff
@@ -37,7 +37,11 @@ class TestLogger implements ITelemetryBufferedLogger {
 		event.testName = this.testName ?? currentTestName;
 		event.testVariant = testVariant;
 		event.hostName = pkgName;
-		this.parentLogger.send({ ...event, ...propsDict });
+		this.parentLogger.send({
+			...event,
+			...propsDict,
+			details: JSON.stringify(process.env.FLUID_LOGGER_PROPS),
```

Comment:
Same as above, I think `JSON.parse()`? And here is where I think we want special casing to see if there's a `hostName` property in the JSON object, and if so, add it next to `details`, so the way that the Performance Benchmarks currently overrides hostName still works.

Reply:
Where is the example where we use `JSON.parse()`? I can only find examples where we use stringify ([here](https://github.com/microsoft/FluidFramework/blob/364d9b5cb745435bc2d31b8adf87459b6a444a5a/packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts#L526) and [here](https://github.com/microsoft/FluidFramework/blob/364d9b5cb745435bc2d31b8adf87459b6a444a5a/packages/test/test-service-load/src/runner.ts#L246), for example.

Reply:
Hhm I might have confused myself. We use JSON.parse() to get the FLUID_LOGGER_PROPS on lines 21-24. But you're right, when adding `details` to an event we stringify them. What we should do though, is reuse `propsDict` instead of getting it from `process.env.FLUID_LOGGER_PROPS` again each time.
- The whole `includes()` check can become `hostName: propsDict.hostName ?? pkgName`
- `details` can just be `propsDict`
- And the `...propsDict` in line 42 I think can go away? Since the only use of FLUID_LOGGER_PROPS seems to be Performance Benchmarks setting the hostname, and we already handled that with the special case logic (which warrants a comment, btw) I think it's ok that for any future uses those props go only in `details`.

Reply:
Ah, that makes more sense. I'll make those changes!